Python/flowgraph.c

```diff
@@ -1517,15 +1517,188 @@ newop_from_folded(PyObject *newconst, PyObject *consts,
     return SUCCESS;
 }
 
+/* Check whether a collection doesn't containing too much items (including
```

Comment:
I know this comment was moved from elsewhere, but we can fix the grammar while we're here:
```suggestion
/* Check whether a collection doesn't contain too many items (including
```